src/quicktype-core/language/Swift.ts

```diff
@@ -98,6 +98,19 @@ class SwiftDateTimeRecognizer extends DefaultDateTimeRecognizer {
     }
 }
 
+export class SwiftProperty {
```

Comment:
This should just be an `interface`, not a `class`.  It just holds data.

Reply:
TIL: Interfaces in typescript